src/components/users/LoginForm.js

```diff
@@ -16,8 +16,16 @@ const LoginForm = () => {
     const { email, password } = inputs;
     // cleanAlert is to remove error alert on change. This will be erased
     const [cleanAlert, setCleanAlert] = useState(false);
+    const [missingEmail, setMissingEmail] = useState(false);
+    const [missingPassword, setMissingPassword] = useState(false);
     // function to update the inputs when the user modifies the input
     function handleChange(e) {
+        const field = e.target.name;
+        if (field === "email") {
```

Comment:
I guess you can omit this piece of code and the one from line 39 to 44. Then, in lines 57 and 60 evaluate inputs' values themselves. If a field is "" it means it's empty, it is not otherwise.